packages/web/src/pages/upload-page/fields/SourceFilesField.tsx

```diff
@@ -71,16 +70,19 @@ export const SourceFilesField = () => {
     useTrackField<Download[typeof ALLOW_DOWNLOAD_BASE]>(ALLOW_DOWNLOAD)
   const [{ value: followerGatedValue }, , { setValue: setFollowerGatedValue }] =
     useTrackField<Download[typeof FOLLOWER_GATED_BASE]>(FOLLOWER_GATED)
+  const [{ value: cid }, , { setValue: setCidValue }] =
```

Comment:
is this field just for passthrough on edit?

Reply:
yeah judging by old form its just always cid, but needed for schema validation on libs side